docs/contributing/triaging-github-issues.md

```diff
@@ -3,7 +3,103 @@ title: Triaging GitHub Issues
 issue: https://github.com/gatsbyjs/gatsby/issues/11810
 ---
 
-This is a stub. Help our community expand it.
+## What is the aim of this document?
 
-Please use the [Gatsby Style Guide](/contributing/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+As a Gatsby core team, we've found patterns that help us effectively triage incoming Github issues, providing answers to the community's questions, identify bugs as well as opportunities to contribute to Gatsby core. Triaging issues is a great way to contribute to the Gatsby community and share your knowledge, without necessarily requiring a lot of deep context about the way the Gatsby codebase works.
+
+We want to share these patterns with the broader community, so that if you're interesting in helping us triage, you're able to do so more effectively!
+
+This document should answer any questions you might have along the way. We’ll answer common questions, list guidelines and illustrate a decision tree.
+
+## What is first touch maintenance?
+
+For Gatsby the first line of communication between a user and the team is the issue tracker on GitHub. Typically, every day 20-30 issues are opened -- that's one every hour!
+
+These issues could be:
+
+- a question that can be answered immediately
+- a bug report
+- a request for a feature
+- or a discussion on a complicated use case
+
+As a first touch maintainer, you might sift through, triage, communicate and manage this first line of communication.
+
+They typically:
+
+- answer questions that can be answered immediately
+  -test and reproduce possible bug reports and label them appropriately
```

Comment:
Check the spacing at the start of this line to make sure the list item renders properly!